modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,25 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* Ensure that you have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
 
 .Procedure
 
-. Create and deploy the MariaDB service to your project:
-.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all available services in the *Developer Catalog*.
-.. Select *Databases* to list all the database services and then click *MariaDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
+. To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group
 
-. To add the MariaDB service to the existing application group, press kbd:[Shift]+ drag the *mariadb* pod to the application group that contains the Node.Js component. The MariaDB service is added to the existing application group.
-
-. Click the MariaDB service node and confirm that the `app.kubernetes.io/part-of=<application-name>` label is in the *Labels* section in the *Overview* Panel. Adding a component to an application group automatically adds the required labels to the component.
+. Dragging a component and adding it to an application group automatically adds the required labels to the component.
 +
 .Application grouping
 image::odc_app_grouping_label.png[]
 
+Alternatively, you can also add the component to an application as follows:
+
+. Click on the service pod to see the *Overview* panel to the right.
+
+. Click the *Actions* drop-down menu on the upper right of the panel and select *Edit Application Grouping*.
```

Comment:
IBM SG: Accessibility and globalization requirement: People who have limited vision, or who use interfaces that are translated into a
bidirectional language, might not understand information if it is conveyed only by location. Provide additional descriptive
information. For example, write "The list of tables is displayed in the Table List pane on the right side of the window", not "The
list of tables is displayed on the right side of the window".

Reply:
Does my sentence not align with this? I'll discuss this also in the Gmeet.

Reply:
I'm not sure - I don't know what the application looks like. If "the *Actions* drop-down menu" is easy to find without mentioning the upper-right area of the panel, then probably no change is needed. If you can name an additional landmark, it might be helpful.

Reply:
Got it, thanks.